### PR TITLE
[Zowe V2] Change default ports to be in the 7550+ range to align with APIML

### PR DESCRIPTION
**Pull request by bobadesakshi21**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [x] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.
This PR addresses Issue:
- Changing the port number for the servers such that they are consecutive after the APIML ports.
- This makes our port assignments more organized.

:heart:Thank you!

### CODE REVIEW

File: docs/extend/extend-desktop/mvd-dataservices.md [hunk 1]
```diff
@@ -151,7 +151,7 @@ In the `zlux-app-server/zluxserver.json` file, use the example below to specify
           }
         }
       },
-      "portRange": [8545,8600]
+      "portRange": [7558,7613]
```
Comment (1000TurquoisePogs) [inline docs/extend/extend-desktop/mvd-dataservices.md:154]: To be honest it would be good to set this to like, 7600-7650. It's just a range of ports to use, but if we put it to close to the existing ports it could become overlapping by accident and make the documentation outdated.

File: docs/user-guide/systemrequirements-zos.md [hunk 2]
```diff
@@ -110,13 +110,13 @@ The following ports are required for Zowe. These are default values. You can cha
...
-| 8547 | _FILES_API_PORT_ | Port of the service that provides REST APIs to MVS and USS file systems.
-| 8548 | _MVS_EXPLORER_UI_PORT_ | Port of the MVS Explorer GUI for working with data sets in the Zowe Desktop.
-| 8550 | _USS_EXPLORER_UI_PORT_ | Port of the USS Explorer GUI for working with USS in the Zowe Desktop.
-| 8544 | _ZOWE_ZLUX_SERVER_HTTPS_PORT_ | The Zowe Desktop (also known as ZLUX) port used to log in through web browsers.
-| 8542 | _ZOWE_ZSS_SERVER_PORT_ | Z Secure Services (ZSS) provides REST API services to ZLUX, used by the File Editor application and other ZLUX applications in the Zowe Desktop.
+| 7558 | _JOBS_API_PORT_ | Port of the service that provides REST APIs to z/OS jobs used by the JES Explorer.
+| 7560 | _JES_EXPLORER_UI_PORT_ | Port of the JES Explorer GUI for viewing and working with jobs in the Zowe Desktop.
+| 7559 | _FILES_API_PORT_ | Port of the service that provides REST APIs to MVS and USS file systems.
+| 7561 | _MVS_EXPLORER_UI_PORT_ | Port of the MVS Explorer GUI for working with data sets in the Zowe Desktop.
+| 7562 | _USS_EXPLORER_UI_PORT_ | Port of the USS Explorer GUI for working with USS in the Zowe Desktop.
```
Comment (1000TurquoisePogs) [inline docs/user-guide/systemrequirements-zos.md:117]: these ones will be disappearing in v2

File: versioned_docs/version-v1.16.x/extend/extend-desktop/mvd-apptoappcommunication.md [hunk 3]
```diff
@@ -96,7 +96,7 @@ The Cross-launch feature allows you to:
 
 #### Sample URL
 
-``https://localhost:8544/ZLUX/plugins/org.zowe.zlux.bootstrap/web/?app2app=org.zowe.zlux.ng2desktop.webbrowser:launch:create:data:{"url":"https://github.com/zowe/zlux-app-manager/pull/234","enableProxy":true}&app2app=org.zowe.zlux.ng2desktop.webbrowser:message:create:data:{"url":"https://github.com/zowe/zlux-app-manager/pull/234","enableProxy":true}&app2app=org.zowe.zlux.ng2desktop.webbrowser:message:create:org.zowe.zlux.test.action:{"data": {"url":"https://github.com/zowe/zlux-app-manager/pull/234","enableProxy":true}}``
+``https://localhost:7556/ZLUX/plugins/org.zowe.zlux.bootstrap/web/?app2app=org.zowe.zlux.ng2desktop.webbrowser:launch:create:data:{"url":"https://github.com/zowe/zlux-app-manager/pull/234","enableProxy":true}&app2app=org.zowe.zlux.ng2desktop.webbrowser:message:create:data:{"url":"https://github.com/zowe/zlux-app-manager/pull/234","enableProxy":true}&app2app=org.zowe.zlux.ng2desktop.webbrowser:message:create:org.zowe.zlux.test.action:{"data": {"url":"https://github.com/zowe/zlux-app-manager/pull/234","enableProxy":true}}``
```
Comment (1000TurquoisePogs) [inline versioned_docs/version-v1.16.x/extend/extend-desktop/mvd-apptoappcommunication.md:99]: Somehow all the content in versioned_docs got flagged.
Don't make any changes to versioned_docs. Revert these changes.
Because, they're old versions, so the old ports are still correct for those.

**Review by 1000TurquoisePogs — CHANGES_REQUESTED** [review]
Mostly agree but I don't understand what we should do with datasets.json and jobs.json
I'm not sure if the APIs will exist in v2, but if they do they would be accessed from the port 7556 but would have some new URL prefix.
Also, we can't edit the old docs, just the current one.

This needs to go on a v2 branch anyway, so no worry about urgency.

**Comment by nannanli** [comment]
@bobadesakshi21 Doc updates for Zowe V2 will need to go to the `v2-doc-branch` before V2 is live. Would you please change the base branch to `v2-doc-branch`? Thanks!
